boards/arm64/nxp_ls1046ardb/doc/index.rst

```diff
@@ -0,0 +1,128 @@
+.. _nxp_ls1046ardb:
+
+NXP LS1046A RDB
+#################################
+
+Overview
+********
+
+The LS1046A reference design board (RDB) is a high-performance computing,
+evaluation, and development platform that supports the Layerscape LS1046A
+architecture processor. The LS1046ARDB board supports the Layerscape LS1046A
+processor and is optimized to support the DDR4 memory and a full complement
+of high-speed SerDes ports.
+
+The Layerscape LS1046A processor integrates four 64-bit Arm(R) Cortex(R) A72
+cores with packet processing acceleration and high-speed peripherals. The
+impressive performance of more than 32,000 CoreMarks, paired with 10 Gb
+Ethernet, PCIe Gen. 3, SATA 3.0, USB 3.0 and QSPI interfaces provides an
+excellent combination for a range of enterprise and service provider
+networking, storage, security and industrial applications.
+
+Hardware
+********
+
+LS1046A RDB boards supports the following features:
+
+
+- Four 32/64-bit Arm(R) Cortex(R)V8 A72 CPUs, up to 1.6 GHz core speed
+- Supports 8 GB DDR4 SDRAM memory
+- SDHC port connects directly to an adapter card slot, featuring 4 GB eMMCi
+  memory device
+- One 512 MB SLC NAND flash with ECC support (1.8 V)
+- CPLD connection: 8-bit registers in CPLD to configure mux/demux selections
+- Support two 64 MB onboard QSPI NOR flash memories
+- USB:
+  - Two USB 3.0 controllers with integrated PHYs.
+  - One USB1 3.0 port is connected to a Type A host connector.
+  - One USB1 3.0 port is configured as On-The-Go (OTG) with a Micro-AB connector.
+  - One USB2.0 is connected to miniPCIe connector .
+- Ethernet:
+  - Supports SGMII 1G PHYs at Lane 2 and Lane 3
+  - Supports SFP+module with XFI retimers
+  - Supports AQR106/107 10G PHY with XFI/2.5G SGMII
+- PCIe and SATA:
+  - Mini PCIe express x1 (Gen1/2/3)card
+  - Standard PCIe x1 (Gen1/2/3) card
+  - Standard PCIe x1 (Gen1/2/3) card
+  - One SATA 3.0 connector
+
+Supported Features
+==================
+
+NXP LS1046A RDB board default configuration supports the following
+hardware features:
+
++-----------+------------+--------------------------------------+
+| Interface | Controller | Driver/Component                     |
++===========+============+======================================+
+| GIC-400   | on-chip    | GICv2 interrupt controller           |
++-----------+------------+--------------------------------------+
+| ARM TIMER | on-chip    | System Clock                         |
++-----------+------------+--------------------------------------+
+| UART      | on-chip    | NS16550 compatible serial port       |
++-----------+------------+--------------------------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+
+The default configuration can be found in the defconfig file for NON-SMP:
+
+        ``boards/arm64/nxp_ls1046ardb/nxp_ls1046ardb_defconfig``
+
+Or for SMP:
+
+	``boards/arm64/nxp_ls1046ardb/nxp_ls1046ardb_smp_defconfig``
```

Comment:
Do you want to remove this if you are going to push SMP at a later time?

Reply:
Done, I removed SMP related document, please review again, thanks.